app/views/play/level/modal/CourseVictoryModal.coffee

```diff
@@ -166,10 +167,11 @@ module.exports = class CourseVictoryModal extends ModalView
       nextLevelOriginal = findNextLevelsBySession(@levelSessions.models, classroomLevels)
     else if @campaign # fetch next based on course's campaign levels (for teachers)
       currentLevel = @campaign.levels[@level.get('original')]
-      # TODO how to get current level stage for capstone and load capstone from the next stage, if there is no level session
-      # if (currentLevel.isPlayedInStages)
-      #   currentLevelStage = ?
-      nextLevelOriginal = (getNextLevelForLevel(currentLevel) || {}).original
+      if (currentLevel.isPlayedInStages && @capstoneStage) # @capstoneStage comes from PlayLevelView's query params
```

Comment:
If I remember the system correctly it's impossible for the capstoneStage to be the number `0` (which would be falsy). Is this invariant a correct assumption?

Reply:
That is correct, in case the query param sets capstoneStage = 0, we will treat it as capstone stage 1. I will add a commit for the default value of capstone stage in `getNextLevelForLevel`